testsuite/forge/src/interface/swarm.rs

```diff
@@ -45,6 +46,8 @@ pub trait Swarm {
     /// Removes the FullNode with the provided NodeId
     fn remove_full_node(&mut self, id: NodeId) -> Result<()>;
 
+    fn cluster(&mut self) -> Result<Cluster>;
+
```

Comment:
I don't think we should include this method as apart of this trait. I would expect that everything using the forge interfaces would use the Swarm trait and not the Cluster type. I do understand that the TxEmitter currently requires that it operates on a `Cluster` though, so perhaps in the test you've added you can construct one temporarily, but it shouldn't be added to this trait.